pkg/attestlib/jwt_test.go

```diff
@@ -96,10 +96,69 @@ func TestVerifyJWT(t *testing.T) {
 				}
 			} else {
 				if err != nil {
-					t.Errorf("Unexpected error: %e", err)
+					t.Errorf("Unexpected error: %v", err)
 				}
 
 			}
 		})
 	}
 }
+
+func TestNewJwtSigner(t *testing.T) {
+	tcs := []struct {
+		name          string
+		key           []byte
+		publicKeyId   string
+		alg           SignatureAlgorithm
+		expectedError bool
+	}{
+		{
+			name:          "new jwt singer success",
+			key:           []byte(rsa2048PrivateKey),
+			publicKeyId:   "kid",
+			alg:           RsaSignPkcs12048Sha256,
+			expectedError: false,
+		},
+		{
+			name:          "new jwt singer with no key id success",
+			key:           []byte(rsa2048PrivateKey),
+			publicKeyId:   "",
+			alg:           RsaSignPkcs12048Sha256,
+			expectedError: false,
+		},
+		{
+			name:          "new jwt singer with bad key fails",
+			key:           []byte("some-key"),
+			publicKeyId:   "",
+			alg:           RsaSignPkcs12048Sha256,
+			expectedError: true,
+		},
+	}
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			_, err := NewJwtSigner(tc.key, tc.alg, tc.publicKeyId)
+			if tc.expectedError {
+				if err == nil {
+					t.Errorf("NewJwtSigner(...) = nil, expected non nil")
```

Comment:
nit: spacing around =

Also ditto as above, would this be confusing as to which return value it is referring to?

Reply:
done.